python/tests/unit/arcticdb/version_store/test_empty_column_type.py

```diff
@@ -498,11 +499,11 @@ def test_float(self, lmdb_version_store_static_and_dynamic, float_dtype):
         )
         lmdb_version_store_static_and_dynamic.update(
             "sym",
-            pd.DataFrame({"col": [None, np.nan]}, index=self.update_index(), dtype=float_dtype)
+            pd.DataFrame({"col": [None, None]}, index=self.update_index())
         )
         assert_frame_equal(
             lmdb_version_store_static_and_dynamic.read("sym").data,
-            pd.DataFrame({"col": [1, np.nan, np.nan, 4]}, index=self.index(), dtype=float_dtype)
+            pd.DataFrame({"col": [1, float("NaN"), float("NaN"), 4]}, index=self.index(), dtype=float_dtype)
```

Comment:
Why?

Reply:
Artifact from testing. Changed to `np.nan, float("NaN")` so that we cover both.